app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FoodsController < ApplicationController 
+  def index
+    @query = params[:q]
+    key = "X5MqWQCLRSlxUlXnpdm2q51W13hIhOhj5krsN0ep"
+    conn = Faraday.new(url: "https://api.nal.usda.gov") do |faraday|
+      faraday.headers['api_key'] = "X5MqWQCLRSlxUlXnpdm2q51W13hIhOhj5krsN0ep" #ENV['usda_api_key']
+    end
+    
+    response = conn.get("/fdc/v1/foods/search?query=#{@query}&numberOfResultsPerPage=10&api_key=#{key}")
+
+    json = JSON.parse(response.body, symbolize_names: true)
+    
+    @num_results = json[:totalHits]
+    
+    @foods = json[:foods].map do |food_data|
```

Comment:
Would love to see you scope this to the `.first(10).map...` so that we only ever create 10 Food objects instead of thousands. This currently causes a performance issue.